engine/src/cmd/collide2/opcodetypes.h

```diff
@@ -171,7 +171,7 @@ typedef int ptrdiff_t;
 #define _PTRDIFF_T_DEFINED
 #endif
 
-#if !defined(CS_HAVE_INTMAX_T)
+#if !defined(CS_HAVE_INTMAX_T) && !__APPLE__
```

Comment:
shouldn't that be `!defined(__APPLE__)`?

Reply:
The two expressions are probably functionally equivalent. I.e., when `__APPLE__` is defined, it is generally defined to the value `1` or something similar. I saw code elsewhere on the web that used the type of expression I used here, and concluded that perhaps that is the more conventional or standard way to do it. But I am not 100% certain of that. If you like, I can certainly switch back to the `!defined(__APPLE__)` syntax we have used previously.

Reply:
We use `!defined(__APPLE__)` elsewhere and should remain internally consistent at the very least; I think it's also clearer from a C Macro perspective as it's not required that these macros carry a value.

It's also consistent on the line since the first parameter is `!defined()` too.

Reply:
Done ✔️